mypy/stubdoc.py

```diff
@@ -24,7 +24,7 @@
 class ArgSig:
     """Signature info for a single argument."""
 
-    _TYPE_RE = re.compile(r'^[a-zA-Z_][\w\[\], ]*(\.[a-zA-Z_][\w\[\], ]*)*$')
+    _TYPE_RE = re.compile(r'^[a-zA-Z_][\w\[\], ]*(\.[a-zA-Z_][\w\[\], ]*)*$')  # type: Final
```

Comment:
Just curious, what was the warning here?

Reply:
"Unsupported default attribute value", like for the other one